infra/bootstrap.sh

```diff
@@ -64,13 +64,30 @@ EOM'
 
 sudo service google-cloud-ops-agent restart
 
-sudo su - validator -c 'git clone https://github.com/tablelandnetwork/go-tableland.git /home/validator/go-tableland'
-sudo su - validator -c 'cat /tmp/.env_validator > /home/validator/go-tableland/docker/deployed/testnet/api/.env_validator'
-sudo su - validator -c 'cat /tmp/.env_grafana > /home/validator/go-tableland/docker/deployed/testnet/grafana/.env_grafana'
-sudo su - validator -c 'cat /tmp/.env_healthbot > /home/validator/go-tableland/docker/deployed/testnet/healthbot/.env_healthbot'
-sudo su - validator -c 'cat /tmp/grafana.db > /home/validator/go-tableland/docker/deployed/testnet/grafana/data/grafana.db'
 
-#sudo su - validator -c 'cd ~/go-tableland/docker && make testnet-up'
+sudo -u validator  -i bash <<-'EOF'
+export TBLENV=$(uname -n | cut -d '-' -f 3)
+
+git clone https://github.com/tablelandnetwork/go-tableland.git ~/go-tableland
+cd ~/go-tableland && git checkout jsign/testnetmainnetsplit
+
+cat /tmp/.env_validator > ~/go-tableland/docker/deployed/${TBLENV}/api/.env_validator
+cat /tmp/.env_grafana > ~/go-tableland/docker/deployed/${TBLENV}/grafana/.env_grafana
+cat /tmp/.env_healthbot > ~/go-tableland/docker/deployed/${TBLENV}/healthbot/.env_healthbot
+cat /tmp/grafana.db > ~/go-tableland/docker/deployed/${TBLENV}/grafana/data/grafana.db
+
+mkdir ~/.ssh && chmod 700 ~/.ssh
+echo "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEehs1xfBMLKpwV4sAIko++GQgauYXf5SNY4tl9ArTOG ops@textile.io" > ~/.ssh/authorized_keys
+
+crontab - <<-MOF
```

Comment:
Adding also the cron tasks.

I used an extra nested `<<-`  (`MOF`, since `EOF` is used in the parent)  to make it easy to add further crontabs in the future without having to do nasty escaping.